docs/usage/custom-dns.md

```diff
@@ -0,0 +1,51 @@
+# Custom DNS Configuration
+
+Gardener provides Kubernetes-Clusters-As-A-Service where all the system components (e.g., kube-proxy, networking, dns, ...) are managed. 
+As a result, Gardener needs to ensure and auto-correct additional configuration to those system components to avoid un-necaessary down-time. 
```

Comment:
```suggestion
As a result, Gardener needs to ensure and auto-correct additional configuration to those system components to avoid unnecessary down-time.
```